### PR TITLE
Kenna Rearick

### PR DESCRIPTION
**Pull request by knnar**
I forked the original hw01 repo and then cloned it to my own GitHub account. I edited the README.md file in the RStudio console, and then used the RStudio Git tab to review the differences, describe my changes in the "commit" box, and push my revision back to GitHub. Finally, I navigated back to this original repo, where I created a pull request with my revised README.md. I found the "Homework Workflow" page, the GitHub forking tutorial page, and the "Mastering Markdown" tutorial page to be particularly helpful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# Biography
+
+My name is **Kenna Rearick**. I am currently attending the University of Chicago, where I am a **graduate student** in the **Social Service Administration** Advanced Standing Program. I am also a research assistant intern at the University of Chicago **Urban Labs**, where I assist with the Supportive Release Center research study. 
+
```
Comment (yuhan-jin) [inline README.md:4]: good use of header and emphasis like bold.

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+Prior to attending UChicago, I graduated from Youngstown State University with my **Bachelor's in Social Work**. My prior experience includes frontline work with children and families who have experienced homelessness, domestic violence, and other forms of trauma. I am also a proud [AmeriCorps NCCC](https://www.nationalservice.gov/programs/americorps/americorps-programs/americorps-nccc) alum. During my time in AmeriCorps, I traveled through more than 20 states and I lived in the following locations:
+
```
Comment (yuhan-jin) [inline README.md:6]: good biography but would it be better to rearrange the paragraph into different sections. For example, seperating the link from your body part.

File: README.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+* Baltimore, MD
+* Washington, DC
+* New York, NY
+* Seattle, WA
+* Stayton, OR
+
+I currently live in Hyde Park with my cat, Venus. ![Venus](https://i.imgur.com/9XmLxVp.jpg)
```
Comment (yuhan-jin) [inline README.md:13]: good use of bullet point and I enjoy your picture. Your assignment satisifies all the requirements. Good job! The only suggestion I would like to say: trying to segment your body paragraphy so as to make your statement in an organized way.

File: README.md [hunk 4]
```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# Biography
+
+My name is **Kenna Rearick**. I am currently attending the University of Chicago, where I am a **graduate student** in the **Social Service Administration** Advanced Standing Program. I am also a research assistant intern at the University of Chicago **Urban Labs**, where I assist with the Supportive Release Center research study.
```
Comment (frania1) [inline README.md:3]: Yay SSA! You got your bold text requirement.

File: README.md (hunk 2 above)
Comment (frania1) [reply README.md:6]: Link requirement fulfilled. Trauma work is important! I personally liked how you embedded a link into your paragraph.

File: README.md (hunk 3 above)
Comment (frania1) [reply README.md:13]: Bullets and link fulfilled!

**Review by frania1 — APPROVED** [review]
Great job! All the requirements were fulfilled and your cat is adorbz. I was trying to figure out how to get bullets on my code, and your code helped answer that question, so thank you for that!

**Review by frania1 — APPROVED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |      x     |              |            |

File: README.md (hunk 3 above)
Comment (rrodriguezbarron) [inline README.md:13]: I am NOT a cat person but this is a **beautiful** cat.

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |           |    x          |            |
| **Achievement, creativity** |        x   |              |            |
| **Ease of access**          |           |   x           |            |

Remarks:

* Great job on the URL and list
* The link to the webpage and image both worked correctly

**Pull request closed without merging**